### DISCUSSION TITLE
Setup on different machines

### DISCUSSION THREAD

**Discussion by Jens-Wymeersch** [mKeRix/room-assistant#464] [General]
Dear all,

I want to start with this project in my house as I have multiple devices that I use for home automation. This will still grow in the future, but before I'm starting I want to have a view where I'm going.
- I have a Windows i5 on which I'm running multiple VMs with virtualbox. Would RA work on such an environment or do I really need to run a Docker image natively on windows ?
- I have Home Assistant running on Proxmox VM. Any idea how I get the HA version to actually work ? I assume the problem is that I can't get the data from the BT through to the VM. 
- I intend to have a RPI Kodi/OpenElec. Can I get somehow RA on this one ? If so, how would this work ?
- The other 2 RPIs are LT OS and full RasperianOS. It should be quite standard here. Correct ?

**Comment by mKeRix** [comment]
room-assistant should work on a VM - in general it can run anywhere you can run NodeJS or Docker, which includes VMs. You need to ensure that the interfaces for whatever integrations you are planning to use are passed through properly of course. For Bluetooth it would for example be important that you can see an hci adapter on the VM (check `sudo hciconfig`).

For the others:
- Proxmox -> no clue here unfortunately, haven't used it
- OpenELEC -> based on their homepage I would say this will be difficult, as that is a very slimmed down version of Linux (although not impossible if you figure out which packages to install)
- Raspbian (or any other normal Linux distribution) are the standard, yes
